ctyparser/bigcty.py

```diff
@@ -92,32 +114,56 @@ def import_dat(self, dat_file: Union[str, os.PathLike]) -> None:
         with dat_file.open("r") as file:
             cty_dict = dict()
 
+            # get the version from the file, set filepointer to beginning of file afterwards
             ver_match = re.search(self.regex_version_entry, file.read())
             self._version = ver_match.group(1) if ver_match is not None else ""
             file.seek(0)
 
+            # variable to store last DXCC introduction with countryname, zones, etc. for further use
             last = ''
+            
             while True:
+                # read a line
                 line = file.readline().rstrip('\x0D').strip(':')
```

Comment:
"read a line" is not a descriptive comment at all. saying something like this is better. I also changed `\x0D` to `\r` for code clarity.

```suggestion
                line = file.readline().rstrip('\r').strip(':')  # remove unnecessary carriage returns and colons
```